pkg/volume/local/local.go

```diff
@@ -50,6 +52,11 @@ const (
 
 func (plugin *localVolumePlugin) Init(host volume.VolumeHost) error {
 	plugin.host = host
+	plugin.volumeLocks = keymutex.NewKeyMutex()
+	mntTracker = localVolumeMounts{
```

Comment:
That will require refactoring of GetMountRefs() which might impact other parts of code which use it.. I will look into it.